Ajout modele gemini

Moucharraf opened the pull request:
J'ai une route pour la génération de test avec le modèle Gemini

Moucharraf committed:
Ajout modele gemini

PierreLouisLetoquart reviewed (commented):
Est-ce que tu peux juste enlever tous les fichiers autre que `geminy.py` et rajouter un `backend/README.md` qui explique comment lancer le backend et qu'est-ce qu'il contient?

Moucharraf committed:
Rectificatif

Ajout des instructions dans le README et renommage du fichier Gemini.py en gemini.py
